etc/scripts/release.hs

```diff
@@ -206,7 +212,11 @@ rules global@Global{..} args = do
                 ,[" --haddock" | gTestHaddocks]
                 ,[" stack"]]
             let cmd' c = cmd (AddPath [tmpDir] []) stackProgName (stackArgs global) c
-            () <- cmd' "test" gBuildArgs integrationTestFlagArgs "--pedantic --exec stack-integration-test stack"
+            () <- cmd
```

Comment:
It looks like this will skip the unit tests, and only run the integration tests.  Is that intended?

Reply:
Semi-intentional. I didn't want to run the tests in this step, but I thought they were run in the previous step. I'll modify the code so it runs the tests above.